styles/globals.css

```diff
@@ -234,4 +237,19 @@
     list-style: decimal;
     list-style-position: inside;
   }
+
+  #oasDefer a,
+  #oasLink0,
```

Comment:
would it work with just a.generatedLink ?, I think so

Reply:
The links in the result card don't have this class